packages/@eth-optimism/configs/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
this file needs to be in packages/configs not packages/@eth-optimism/configs

Reply:
there is no file configs in packages. can i make configs and add tsconfig.json to it.

Reply:
Yes exactly! The goal of this pr is to make the configs package and put the configs in it. Exactly as the first line of your pr descriptions says: 

> Create a new package @eth-optimism/configs in packages/configs